Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -503,11 +503,16 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
         // Dark blurred background.
         blurView.autoPinEdgesToSuperviewEdges()
 
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .leading)
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+        leaveCallViewButton.autoPinEdge(toSuperviewEdge: .leading)
```

Comment:
With the padding already in the image view, pinning to the margin moved the back button too far in.